ts/components/ui/cards/payment/PaymentCardBig.tsx

```diff
@@ -79,7 +79,7 @@ const BigPaymentCardTopSection = (props: PaymentCardStandardProps) => {
       return (
         <View style={IOStyles.flex}>
           <NewH6 style={{ textTransform: "capitalize" }}>
-            {`${props.cardIcon} •••• ${props.hpan}`}
+            {`${props.cardIcon} •• ${props.hpan}`}
           </NewH6>
```

Comment:
We should use `H3` here, not `H6`

Reply:
it does look a tad bit too big IMO, but yes, it more strictly follows design.
adressed, nice catch 👍